go/test/endtoend/backup/vtbackup/backup_only_test.go

```diff
@@ -294,17 +300,18 @@ func resetTabletDirectory(t *testing.T, tablet cluster.Vttablet, initMysql bool)
 
 func tearDown(t *testing.T, initMysql bool) {
 	// reset replication
+	for _, db := range []string{"_vt", "vt_insert_test"} {
```

Comment:
I feel like this should be a variable, so that creation and deletion are kept in sync if and when these values change.

Reply:
not used anywhere else so we should be good here.